docs/UpgradeGuide.md

```diff
@@ -2,57 +2,78 @@
 
 # From 0.x to 1.0
 
-- [Namespacing & the public API](#namespacing--the-public-api)
-- [Configuration](#configuration)
-- [Instrumentation](#instrumentation)
-- [Full list of breaking changes](#full-list-of-breaking-changes)
+Upgrading `ddtrace` from 0.x to 1.x introduces some breaking changes which are outlined below.
 
-Upgrading `ddtrace` from 0.x to 1.x introduces some changes to namespacing, the public API, and the underlying trace data structure.
+**How to upgrade basic usage**
 
-Here's a list of the most common changes you may encounter:
+For users with a basic implementation (configuration file + out-of-the-box instrumentation), only minor changes to your configuration file are required: most applications take just minutes to update. Check out the following sections for a step-by-step guide.
```

Comment:
I'm still quite unconvinced with the use of the word _basic_. Since you didn't like my suggestion of _standard_, I've gotten my thesaurus out and here's a few more suggestions: base, common, simple, typical, regular, ~bourgeois~, ~philistine~.

All of them I think don't carry the weird connotations that sometimes get attached to _basic_ 😄